chatterbot/utils/module_loading.py

```diff
@@ -1,4 +1,5 @@
-import importlib
+from importlib import import_module as import_string
+from inspect import isclass
```

Comment:
If you aren't hitting the issue on Python 3, it might be an issue with relative imports vs absolute imports.